src/Symfony/Component/Form/Console/Descriptor/Descriptor.php

```diff
@@ -113,6 +102,26 @@ protected function collectOptions(ResolvedFormTypeInterface $type)
         $this->extensions = array_keys($this->extensions);
     }
 
+    protected function getOptionDefinition(OptionsResolver $optionsResolver, $option)
+    {
+        $refObject = new \ReflectionObject($optionsResolver);
+        foreach (array('defaults', 'lazy', 'allowedTypes', 'allowedValues', 'normalizers') as $name) {
+            $property = $refObject->getProperty($name);
```

Comment:
Re-thinking about it, I'm a bit worried about relying on internals of another component, even for debug purpose. If one of these properties change in the future, the `debug:form` command will fail.
I see 3 options so far:
1. Keep using reflection from the command side, but be failure proof, for instance by catching the exception if the property does not exist and output a degraded version.
2. Introduce something like a `OptionResolverDebugger` that will be maintain along the `OptionResolver` component itself.
3. Add required getters to the `OptionResolver`, even if there is probably no other use cases appart from debugging right now...

Reply:
Option 2 looks the safest to me if we don't want to introduce these getters to the OptionResolver API.

@Tobion : As you're the merger for the `OptionResolver` component, I'd like to get your opinion on this. Thanks :)

Reply:
> OptionResolverDebugger

this would access private state from OptionResolver, but would sit in the same component, so would be have a friendship relationship? If that's what you mean, I'm :+1: also.

Reply:
That's what I meant. I'll push the code this evening.